openshift/_python36.dc.json

```diff
@@ -192,14 +192,14 @@
       "apiVersion": "v1",
       "kind": "PersistentVolumeClaim",
       "metadata": {
-        "name": "mds-document-storage${SUFFIX}",
+        "name": "mds-document-block-storage${SUFFIX}",
         "annotations": {
-          "volume.beta.kubernetes.io/storage-class": "gluster-file",
+          "volume.beta.kubernetes.io/storage-class": "gluster-block",
           "volume.beta.kubernetes.io/storage-provisioner": "kubernetes.io/glusterfs"
         }
       },
       "spec": {
-        "accessModes": ["ReadWriteMany"],
+        "accessModes": ["ReadWriteOnce"],
```

Comment:
we have more than one backend pod. I don't think we can do this for the backend (we also loose rolling deploys.